cli/waiter/subcommands/maintenance.py

```diff
@@ -0,0 +1,80 @@
+import requests
+
+from waiter import terminal, http_util
+from waiter.token_post import process_post_result, post_failed_message
+from waiter.querying import get_token, query_token
+from waiter.util import guard_no_cluster, logging, print_info
+
+
+def get_cluster_with_token(clusters, token_name):
```

Comment:
Nitpick: `util` feels a good location for this function

Reply:
I put it in `querying` instead because it uses a function in `querying` and I wanted to avoid possible circular dependencies with `util` importing other functions